src/server/pkg/deploy/cmds/cmds.go

```diff
@@ -732,9 +732,10 @@ If <object store backend> is \"s3\", then the arguments are:
 
 			if namespace == "" {
 				kubeConfig := config.KubeConfig(nil)
+				var err error
 				namespace, _, err = kubeConfig.Namespace()
 				if err != nil {
-					log.Errorf("couldn't load kubernetes config (using \"default\"): %v", err)
+					fmt.Fprintf(os.Stderr, "WARNING: using namespace \"default\" (couldn't load namespace from kubernetes config: %v)\n", err)
```

Comment:
Why this change? I'm trying to move us over to use logging on the client-side more

Reply:
There was some confusion in slack when Nitin was running the manifest test locally to understand the fix, and saw errors of the form `ERRO[0000] couldn't load kubernetes config (using "default"): invalid configuration: no configuration has been provided`. I had to go read the code to realize that the error was benign, so this was just me trying to make the error clearer. Specifcally:
1. I figured we shouldn't have an `ERRO[0000]` prefix, since that makes this look like a hard error, which it isn't
2. It now says `using namespace "default"`, which it didn't before ("default what?"). It also says it at the beginning, so it's clear what this error message is about
3. It prints to stderr so that it doesn't mess up this command's output.

I do think this is a clearer error that what we have (which I think is a low bar), but I cannot overstate my disinterest in dying on this hill, so if you want I can rebase this PR and drop this commit if you have a different way you want to fix this. Just let me know what you want. As well, if you overwrite this change in the next PR—great!